site/_posts/2017-10-02-Exploring-Budapest-while-the-others-are-working.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Last Week in ManageIQ: Exploring Budapest while the others are working"
+author: skateman
+date: 2017-10-02
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello everyone, my name is [Dávid Halász](https://github.com/skateman) and I'd like to guide you through the changes we made in the last week.
+
+## Featured
+
+During the weekend [Roman](https://github.com/romanblanco), [Julian](https://github.com/juliancheal) and myself were attending at [EuRuKo 2017](https://www.euruko2017.org/) in Budapest. It is a beautiful city where you can among others eat [halászlé](https://hu.wikipedia.org/wiki/Hal%C3%A1szl%C3%A9) (not named after me) and see the [Halászbástya](https://hu.wikipedia.org/wiki/Hal%C3%A1szb%C3%A1stya) (also not named after me).
+
+![Budapest landscape](https://i.imgur.com/fTSUeue.jpg)
+
+We've heard a lot of interesting presentations and also met with [Yukihiro Matsumoto](https://github.com/matz), the creator of the Ruby language.
+
+![Matz and ManageIQ](https://i.imgur.com/BVYNubg.jpg)
+
+While we were at the conference, the others kept improving ManageIQ and merged more than a hundred PRs. As I wasn't there I couldn't follow all the changes, but now I have an opportunity to check them out while picking the most important ones for this post.
+
+## Improved
+
+While [Ladas](https://github.com/ladas) improved how we [save metrics](https://github.com/ManageIQ/manageiq/pull/15976) and [Tim](https://github.com/imtayadeway) added [event streams](https://github.com/ManageIQ/manageiq/pull/16021) as a new product feature we were exploring the Square of Heroes.
+
+![The Square of Heroes](https://i.imgur.com/UjJGoB9.jpg)
+
+## Fixed
+
+Back in the day, the *Service UI* was called *Self Service* and we missed some places were it should've been renamed. Thanks to [Allen](https://github.com/AllenBW) now it is [fixed and consistent](https://github.com/ManageIQ/manageiq-ui-service/pull/1030) across the repository. As we moved our GTL code to Angular, the policy simulation screen got broken and did not display quadicons but thanks to [Milan](https://github.com/mzazrivec) this is now [fixed](https://github.com/ManageIQ/manageiq-ui-classic/pull/2034). In the meantime we explored the square around the Matthias Church.
```

Comment:
Would be nice to have a link to quadicons - http://manageiq.org/docs/guides/ui/quadicons (for the benefit of those like me who didn't realise what it means :stuck_out_tongue: )